### PR TITLE
Create seeder for User and Visitor model

### PR DESCRIPTION
**Pull request by dwiajik**

### CODE REVIEW

File: seeds/VisitorSeed.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  for (let i = 0; i < 1000; i++) {
+    fakeVisitors.push({
+      name: `${faker.name.firstName()} ${faker.name.lastName()}`,
+      dob: dob(),
+      nric: nric(),
+      membershipId: membershipId(),
+      membershipExpiry: faker.date.between('2012-01-01', '2022-12-31'),
+      remarks: faker.random.words(),
+      otherId: faker.lorem.words(),
+      otherFields: {
```
Comment (haydarai) [inline seeds/VisitorSeed.js]: `otherFields` should be an array

Comment (dwiajik) [reply seeds/VisitorSeed.js]: Ah, my mistake, will fix it

File: seeds/UserSeed.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+module.exports = () => {
+  const fakeUserCount = 10;
+  const fakeUsers = [];
+
+  console.log(`Seeding ${fakeUserCount} users...`);
+
+  for (let i = 0; i < 10; i++) {
+    fakeUsers.push({
```
Comment (haydarai) [inline seeds/UserSeed.js]: consider adding, `tokens` (array), `passwordResetExpires` (date), and `passwordResetToken` (string) to comply with the User model generated from the web app.

Comment (dwiajik) [reply seeds/UserSeed.js]: @haydarai I registered in website, but there is no such things:
```
> db.users.find({email: "dwi.aji.kurniawan@gmail.com"}).pretty()
{
	"_id" : ObjectId("59d8d9c8dd8048422a7fb50a"),
	"updatedAt" : ISODate("2017-10-07T13:42:32.469Z"),
	"createdAt" : ISODate("2017-10-07T13:42:32.469Z"),
	"email" : "dwi.aji.kurniawan@gmail.com",
	"password" : "$2a$10$RnhZjl4XSAqt2gAp4anae.trsxpzmNTwR2Jc8bhwxscRMnKt/VczO",
	"role" : "Administrator",
	"tokens" : [ ],
	"__v" : 0
}
```
Can you help with the examples of them?

Comment (haydarai) [reply seeds/UserSeed.js]: Just realized that `passwordResetExpires` and `passwordResetToken` only added when the user have requested a password reset. So, probably just add `tokens` and you're good to go.

Comment (dwiajik) [reply seeds/UserSeed.js]: @haydarai Do you have example of tokens? Or should I just give it empty array?

Comment (haydarai) [reply seeds/UserSeed.js]: Just use empty array, I'm still not sure what is it used for.

Comment (dwiajik) [reply seeds/UserSeed.js]: will do it, thanks

**Comment by dwiajik** [comment]
@haydarai @kuriakinzeng I have updated this PR

File: seeds/UserSeed.js (hunk 2 above)
Comment (dwiajik) [reply seeds/UserSeed.js]: done ya

**Pull request merged**